clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -115,6 +113,8 @@ New check aliases
 Changes in existing checks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Added `ctime` and `localtime` to clang-tidy.
```

Comment:
Please follow format of other entries in this section.

Reply:
Sure, I have updated the release notes text. Could you please check if it is ok?